src/pages/saksbehandling/sakintro/Sakintro.tsx

```diff
@@ -150,21 +149,16 @@ const NyBehandlingVelger = (props: { sakId: string; klageToggle: boolean; intl:
                 variant="secondary"
                 onClick={(e) => {
                     if (anchorEl) {
-                        setAnchorEl(undefined);
+                        setAnchorEl(null);
                     } else {
                         setAnchorEl(e.currentTarget);
                     }
```

Comment:
Hvis du vil bedre +/- lines så kan jeg anbefale 😄 
`onClick={(e) => setAnchorEl(anchorEl ? null : e.currentTarget)}`

Reply:
haha sure, for din skull! :DD